### PR TITLE
Add the base source code for Ruby 2.5 Runtime

### PR DESCRIPTION
**Pull request by remore**
Related Issue(s):
https://github.com/apache/incubator-openwhisk/pull/3725
https://github.com/remore/openwhisk-runtime-ruby/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,110 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Apache OpenWhisk runtimes for Ruby
+[![Build Status](https://travis-ci.org/remore/openwhisk-runtime-ruby.svg?branch=master)](https://travis-ci.org/remore/openwhisk-runtime-ruby)
```
Comment (rabbah) [inline README.md]: update this link.

File: README.md [hunk 2]
```diff
@@ -1 +1,110 @@
...
+$ANSIBLE_CMD setup.yml
+$ANSIBLE_CMD couchdb.yml
+$ANSIBLE_CMD initdb.yml
+$ANSIBLE_CMD wipe.yml
+$ANSIBLE_CMD openwhisk.yml
+```
+
+Or you can use `wskdev` and create a soft link to the target ansible environment, for example:
+```
+ln -s ${ROOTDIR}/ansible/environments/local ${OPENWHISK_HOME}/ansible/environments/local-ruby
```
Comment (rabbah) [inline README.md]: once the manifest change is made upstream i dont think you need this special environment here and we can simplify here.

File: README.md [hunk 3]
```diff
@@ -1 +1,110 @@
...
+```
+./gradlew install
+```
+
+Using gradle for the ActionContainer tests you need to use a proxy if running on Mac, if Linux then don't use proxy options
+You can pass the flags `-Dhttp.proxyHost=localhost -Dhttp.proxyPort=3128` directly in gradle command.
+Or save in your `$HOME/.gradle/gradle.properties`
+```
+systemProp.http.proxyHost=localhost
+systemProp.http.proxyPort=3128
```
Comment (rabbah) [inline README.md]: hmm why did you need to use a proxy? this isn't generally needed even for mac anymore.

File: tests/src/test/scala/actionContainers/Ruby25ActionContainerTests.scala [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+        |  }
+        |end
+      """.stripMargin.trim,
+      enforceEmptyOutputStream = enforceEmptyOutputStream)
+  }
+
+  override val testLargeInput = {
+    TestConfig("""
+        |def main(args)
+        |  puts 'hello stdout'
```
Comment (rabbah) [inline tests/src/test/scala/actionContainers/Ruby25ActionContainerTests.scala]: dont think test cares about stdout/stderr here.

File: README.md (hunk 1 above)
Comment (remore) [reply README.md]: Updated with 73c0cb7ed64d989795eb0070b825349175ed9aaa

File: README.md (hunk 2 above)
Comment (remore) [reply README.md]: Noted. Which one is corresponding PR for this?

File: README.md (hunk 3 above)
Comment (remore) [reply README.md]: Ah those are just coming from other runtime documentation such as nodejs or php. Maybe better to remove these proxy-related documentations unless I have examined it's working?

File: tests/src/test/scala/actionContainers/Ruby25ActionContainerTests.scala (hunk 4 above)
Comment (remore) [reply tests/src/test/scala/actionContainers/Ruby25ActionContainerTests.scala]: You are right. Fixed this with 73c0cb7ed64d989795eb0070b825349175ed9aaa

File: README.md (hunk 3 above)
Comment (rabbah) [reply README.md]: Shouldn’t be needed here or in other runtimes.

Comment (remore) [reply README.md]: @rabbah Understood. Updated with e009e187152368a89a31d246e6a27c95ccf21821

File: tests/.pydevproject [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (rabbah) [inline tests/.pydevproject]: dont need this file.
